scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -768,3 +781,9 @@ VAL_ 506 Drive_Mode 0 "DRIVE_MODE_1" 1 "DRIVE_MODE_2" 2 "DRIVE_MODE_3" 3 "DRIVE_
 VAL_ 507 Start_Switch 0 "OFF" 1 "ON";
 VAL_ 507 Traction_Control_Switch 0 "OFF" 1 "ON";
 VAL_ 507 Torque_Vectoring_Switch  0 "OFF" 1 "ON";
+VAL_ 600 Clear_Left_Fault 0 "NO" 1 "YES";
+VAL_ 600 Clear_Right_Fault 0 "NO" 1 "YES";
+VAL_ 601 Close_Left_Switch 0 "NO" 1 "YES";
+VAL_ 601 Close_Right_Switch 0 "NO" 1 "YES";
+VAL_ 601 Open_Left_Switch 0 "NO" 1 "YES";
+VAL_ 601 Open_Right_Switch 0 "NO" 1 "YES";
```

Comment:
suggest maintaining enum names similar to the switches above (see `Traction_Control_Switch` for example)

Reply:
done

Reply:
*more direct

Reply:
open and close makes more sense to me for a switch imo